store/loomethdb.go

```diff
@@ -93,46 +104,22 @@ func (b *batch) ValueSize() int {
 }
 
 func (b *batch) Write() error {
-	b.parentStore.lock.Lock()
-	defer b.parentStore.lock.Unlock()
-
-	sort.Slice(b.cache, func(j, k int) bool {
-		return bytes.Compare(b.cache[j].key, b.cache[k].key) < 0
-	})
-
-	for _, kv := range b.cache {
-		if kv.value == nil {
-			b.parentStore.Delete(kv.key)
-		} else {
-			b.parentStore.Put(kv.key, kv.value)
-		}
-	}
+	b.levelDBbatch.WriteSync()
```

Comment:
This wasn't a sync write in the EvmStore, so probably shouldn't be sync here either.